app/src/main/java/com/example/greetingcard/view/LoginScreen.kt

```diff
@@ -33,7 +34,7 @@ fun LoginScreen(navController: NavHostController) {
                 .padding(innerPadding)
         ) {
             Spacer(modifier = Modifier.height(24.dp))
-            Title()
+            TitleH1(title = "Bem vindo(a) à Taqtile!")
```

Comment:
```suggestion
            H1(title = "Bem vindo(a) à Taqtile!")
```

H1 já fica claro o sufciente, acredito.